### DISCUSSION TITLE
Compatible with MS SQL Server 2019?

### DISCUSSION THREAD

**Discussion by dbernstein-pallidus** [sqlalchemy/sqlalchemy#8808] [Usage Questions] [answered]
Hi Everyone, I see that sqlalchemy uses sql server 2017 for CI. Are there any issues with sqlalchemy and the ms sql server 2019 version? I am installing ms sql server and would like to know if it's ok to go ahead with the 2019 version for use with sqlalchemy.

Thanks,
Dave

**Comment by zzzeek** [comment, marked as answer]
There are no known issues with SQL Server 2019.  The MSSQL dialect is used for all varieties of Azure cloud databases as well which are much more different than different SQL Server versions.

**Reply by dbernstein-pallidus** [reply]
Thanks so much for the quick response!
